typed-racket-lib/typed-racket/types/type-table.rkt

```diff
@@ -78,7 +78,7 @@
     [((tc-result1: old-t) (tc-result1: t-t))
      (ret (Un old-t t-t))]
     [((tc-results: old-ts) (tc-results: t-ts))
-     ;; filters don't matter at this point, since only
+     ;; props don't matter at this point, since only
```

Comment:
I think this comment is no longer true, so maybe add a note to that effect while you're here.